docs/reference/configuration.md

```diff
@@ -317,31 +325,31 @@ Defines a comma-separated column names having a unique constraint on the table.
 
 ### `SINK_JDBC_CONNECTION_POOL_TIMEOUT_MS`
 
-Defines a database connection timeout in milliseconds.
+Defines a database connection timeout in milliseconds. Follow this [connectionTimeout](https://github.com/brettwooldridge/HikariCP#frequently-used) for more details on using this configuration.
 
 * Example value: `1000`
 * Type: `required`
-* Default value: `1000`
+* Default value: `30000`
 
 ### `SINK_JDBC_CONNECTION_POOL_IDLE_TIMEOUT_MS`
 
-Defines a database connection pool idle connection timeout in milliseconds.
+Defines a database connection pool idle connection timeout in milliseconds. Follow this [idleTimeout](https://github.com/brettwooldridge/HikariCP#frequently-used) for more details on using this configuration.
```

Comment:
Let's add one statement saying "It needs to have a minimum value of 10000 ms"

Reply:
first, i think we don't need that since it's already explained on the given link. but the chance to read from the given link is less. will update it